week-1-ts-ecosystem/w01e01-typescript-basics/src/main.ts

```diff
@@ -25,15 +17,6 @@ const user: User = {
     no: 23,
   },
 }
-
-function hasAddress(user: User) {
-  return Boolean(user.address)
-}
-
-function hasGivenAge(requiredAge: number) {
-  return (user: User) => user.age >= requiredAge
-}
-
 const isAdult = hasGivenAge(18)
```

Comment:
detal: to bym jeszcze dodał do `userValidators` @gkubiak92 - a tak to git jest :)